### PR TITLE
Feature/jupyter book

### PR DESCRIPTION
**Pull request by SiggyF**
Add jupyter book. 

- [x] add jupyter book deps
- [x] add build script
- [x] github action for book deployment (static website app in azure)

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -64,3 +66,9 @@ def assert_log(simulation_object):
         assert_log(sub_process)
 
     return new_log
+
+
+def test_notebook_path():
+    """test if we can find the notebook path"""
+    notebook_path = openclsim.utils.find_notebook_path()
+    assert "notebook" in str(notebook_path), "notebook should be in the notebook path"
```
Comment (Pietervanhalem) [inline tests/test_utils.py:74]: I guess we are not running the notebooks in the test yet?

**Review by Pietervanhalem — COMMENTED** [review]
Looks good!

**Comment by Pietervanhalem** [comment]
We added the github actions cofig to the VO fork. I dont think that conflicting with this? Should we merge the two efforts?

**Review by Pietervanhalem — APPROVED** [review]

File: tests/test_utils.py (hunk 1 above)
Comment (SiggyF) [reply tests/test_utils.py:74]: The notebooks can be run as separate tests. That's another task. In OpenTNSim we do that in [circleci](https://github.com/TUDelft-CITG/OpenTNSim/blob/master/.circleci/config.yml#L23)

**Comment by SiggyF** [comment]
> We added the github actions cofig to the VO fork. I dont think that conflicting with this? Should we merge the two efforts?

You can have multiple github action files next to each other. This one has a quite specific name.

**Pull request merged**